lgc/builder/DescBuilder.cpp

```diff
@@ -87,31 +87,51 @@ Value *DescBuilder::CreateLoadBufferDesc(unsigned descSet, unsigned binding, Val
       abstractType = ResourceNodeType::DescriptorBufferCompact;
 
     std::tie(topNode, node) = m_pipelineState->findResourceNode(abstractType, descSet, binding);
+    if (!node) {
+      // If we can't find the node, assume mutable descriptor and search for any node.
+      std::tie(topNode, node) = m_pipelineState->findResourceNode(ResourceNodeType::DescriptorMutable, descSet, binding);
+      if (!node) {
+        // We did not find the resource node. Return an undef value.
+        return UndefValue::get(getBufferDescTy(pointeeTy));
```

Comment:
Use `PoisonValue` instead of `UndefValue` -- it's a subtle difference, but LLVM intends to move towards poison generally.

Reply:
Switched to PoisonValue